src/footer.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" rel="stylesheet">
+    <title>Complete Footer</title>
+    <script src="../asset/scripts/translate.js"> </script>
+</head>
+<body>
+       <!-- Nút thay đổi ngôn ngữ -->
+   <button onclick="setLanguage('en')">English</button>
+   <button onclick="setLanguage('vi')">Tiếng Việt</button>
+
+    <footer class="bg-black text-white py-4 px-20">
+        <div class="container mx-auto">
+            <div class="flex flex-wrap justify-between">
+                <!-- Contact Section -->
+                <div class="w-full lg:w-1/4 md:w-1/2 mb-4">
+                    <h6 class="font-bold" data-i18n="footer-contact_us">CONTACT US</h6>
+                    <p class="flex items-center">
+                        <i class="fas fa-map-marker-alt text-lime-500"></i>
+                        <span class="ml-2" data-i18n="footer-address">Floor 4, Hanoi Group Building, No. 442 Doi Can, Cong Vi Ward, Ba Dinh District, Hanoi</span>
+                    </p>
+                    <p class="flex items-center">
+                        <i class="fas fa-phone-alt text-lime-500"></i>
+                        <span class="ml-2" data-i18n="footer-phone"> (04) 6674 2332 - (04) 3786 8904</span>
+                    </p>
+                    <p class="flex items-center">
+                        <i class="fas fa-envelope text-lime-500"></i>
+                        <a href="mailto:Support@bizweb.vn" class="text-lime-500 ml-2" data-i18n="footer-email">Support@bizweb.vn</a>
+                    </p>
+                </div>
+                <!-- Shipping Section -->
+                <div class="w-full lg:w-1/6 md:w-1/2 mb-4">
+                    <h6 class="font-bold" data-i18n="footer-shipping">SHIPPING</h6>
+                    <ul class="list-none">
+                        <li><a href="#" class="text-white hover:text-lime-500" data-i18n="footer-online_shopping">Online Shopping</a></li>
```

Comment:
```suggestion
                        <li><a href="#" class="text-white hover:text-lime-500" data-i18n="footer-online_shopping"></a></li>
```
Vì đã có data i18n rồi, nên giá trị trong thẻ không cần nữa Linh nhé. Tương tự cho những chỗ khác nha